dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceFile.java

```diff
@@ -188,6 +188,9 @@ public String toString() {
         @Column(columnDefinition = "text")
         public String metadata = "";
 
+        @Column(columnDefinition = "text")
+        public String platformVersion = "";
```

Comment:
This should probably be non-nullable if empty string is allowed (easier to code when you know the version is always the empty string rather than null or empty string).